frontend/micro-ui/web/micro-ui-internals/packages/modules/Expenditure/src/utils/createBillUtils.js

```diff
@@ -12,7 +12,7 @@ const createDocObject = (document, docType, otherDocFileName="Others", isActive,
     payload_modal.documentUid = document?.[1]?.['fileStoreId']?.['fileStoreId'];
     payload_modal.key = docType;
     payload_modal.additionalDetails = {
-      fileName : document?.[1]?.['file']?.['name'] ? document?.[1]?.['file']?.['name'] :  documentType?.filter(doc=>doc?.name === docType)?.[0]?.code,
+      fileName : document?.[1]?.['file']?.['name'] ? document?.[1]?.['file']?.['name'] :   document?.[0] ?  document?.[0] : documentType?.filter(doc=>doc?.name === docType)?.[0]?.code,
```

Comment:
Use round braces to find the order of execution

Reply:
@jagankumar-egov converted nested ternary into ifelse for better clarification and sonar issue also resolved